### DISCUSSION TITLE
DaVinci AIO Repetier 0.92.10 stops to print after autoleveling

### DISCUSSION THREAD

**Discussion by megejas** [luc-github/Repetier-Firmware-0.92#406] [Q&A] [answered]
Hello all,
Today I encountered strange problem with my DaVinci AIO. It worked fine several years but now it homes axis, heats up, clean nozzle and do nothing more. Repetier Host show progress and in very fast way. Printer also show progress, react to temperature change from host also. All this happen after I tried to solve bad first layer adhesion problems. I tried zmin evaluation,  leveling, autoleveling. I think I did nothing more. After this it stopped real printing. Maybe I selected something wrong by mistake? Or there is some bug somewhere?

**Comment by luc-github** [comment]
be sure you did not enabled dry run by mistake

**Reply by megejas** [reply]
Hi, maybe you can explain where can I select dry run? I heard something about this mode but I do not know where it can be selected. In DaVinci Repetier or in Repetier host?

**Reply by luc-github** [reply]
it is a debug flag that can be changed in pronter menu

**Comment by megejas** [comment, marked as answer]
After another maybe 5 printer power cycles, after reslicing with Slic3r it started to work as before. So all OK now and I do not know what the problem was there.

**Reply by luc-github** [reply]
too many possibilities sorry, hope it won't happen again
